cloud

```diff
@@ -1509,7 +1514,6 @@ function cloud_functions_withdraw() {
     cloud_function_withdraw "$project" "${prefix}pick_notifications"
     cloud_function_withdraw "$project" "${prefix}send_notification"
     cloud_function_withdraw "$project" "${prefix}spool_notifications"
-    cloud_function_withdraw "$project" "${prefix}cache_urls"
```

Comment:
We shouldn't be removing the removal of this function in this commit.

Instead we should be passing the name of the function here via the parameters and removing **that**.